internal/e2e/reference_types_directive/BUILD.bazel

```diff
@@ -21,6 +21,11 @@ ts_library(
 
 jasmine_node_test(
     name = "test",
-    deps = [":test_lib"],
-    node_modules = "//:node_modules",
+    deps = [
+      "@npm//:jasmine",
+      "@npm//:which",
+      # @npm//:which this is needed as the 'exit' package doesn't
```

Comment:
at some point this comment will have to change so that ts_auto_deps understands not to prune the unused dependency.

We had also once discussed being able to add these undeclared dependencies in the `yarn_install` rule so the `@npm//:exit` package definition would get `which` added. Otherwise you have to duplicate this comment in many tests?

Reply:
test if `@npm//:which` is still needed now that `node_modules` files are filtered out of `jasmine_node_test`